api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -791,4 +791,31 @@ public void getCountOfDrugs_shouldReturnTheTotalNumberOfMatchingNumbers() {
 		assertEquals(new Integer(1),
 		    conceptService.getCountOfDrugs(phrase, conceptService.getConcept(conceptId), true, true, true));
 	}
+
+	/**
+	 * @see ConceptServiceImpl#saveConceptProposal(ConceptProposal) 
+	 */
+	@Test
+	public void saveConceptProposal_shouldReturnTheSavedConceptProposalObject() {
+		ConceptProposal conceptProposal = new ConceptProposal();
+		conceptProposal.setOriginalText("OriginalText");
+		List<ConceptProposal> existingConceptProposals = conceptService.getConceptProposals("OriginalText");
+		assertTrue(existingConceptProposals == null || existingConceptProposals.isEmpty());
+		ConceptProposal savedConceptProposal = conceptService.saveConceptProposal(conceptProposal);
+		assertNotNull(savedConceptProposal);
+	}
+
+	/**
+	 * @see ConceptServiceImpl#saveConceptProposal(ConceptProposal)
+	 */
+	@Test
+	public void saveConceptProposal_shouldReturnTheUpdatedConceptProposalObject() {
+		ConceptProposal conceptProposal = new ConceptProposal();
+		conceptProposal.setOriginalText("OriginalText");
+		ConceptProposal savedConceptProposal = conceptService.saveConceptProposal(conceptProposal);
+		assertEquals("OriginalText", savedConceptProposal.getOriginalText());
```

Comment:
No need for this assertion in my opinion here. This should be checked in the other test.